Firestore/Source/Public/FirebaseFirestore/FIRFirestore.h

```diff
@@ -121,10 +131,15 @@ NS_SWIFT_NAME(Firestore)
  */
 - (void)setIndexConfigurationFromJSON:(NSString *)json
                            completion:(nullable void (^)(NSError *_Nullable error))completion
-    NS_SWIFT_NAME(setIndexConfiguration(_:completion:));
+    NS_SWIFT_NAME(setIndexConfiguration(_:completion:)) DEPRECATED_MSG_ATTRIBUTE(
+        "Instead of creating cache indexes manually, consider using "
+        "`PersistentCacheIndexManager.enableIndexAutoCreation()` to let the SDK decide whether to "
+        "create cache indexes for queries running locally.");
```

Comment:
@sunmou99, it'd be a nice-to-have if API diff tool  caught deprecations as well. Here, the ObjC and Swift API were both deprecated.